docs/tutorials/downgrade-extension.md

```diff
@@ -36,29 +36,27 @@ kind: ClusterExtension
 metadata:
   name: example-extension
 spec:
+  namespace: argocd
+  serviceAccount:
+    name: argocd-installer
   install:
     preflight:
       crdUpgradeSafety:
-        policy: Disabled
-    namespace: argocd
-    serviceAccount:
-      name: argocd-installer
+        enforcement: None
   source:
     sourceType: Catalog
     catalog:
       packageName: argocd-operator
       version: 0.6.0
-      upgradeConstraintPolicy: SelfCertified
```

Comment:
It seems that we have more places to change regards this one: https://github.com/search?q=repo%3Aoperator-framework%2Foperator-controller++upgradeConstraintPolicy+&type=code

Reply:
IHMO that would be a commit a PR - Replace upgradeConstraintPolicy with X because Z
It is very wirdo split things in this way.
If we need to revert them how do we do it? 
Honest, I really dislike this approach

Reply:
IHMO that would be a commit a PR - Replace upgradeConstraintPolicy with X because Z
It is very wirdo split things in this way.
What we will do if for any reason we need to revert ALL or partial the API changes?

Reply:
IHMO,  this PR should be at least ideally a follow-up to PR X, addressing all impacted areas across the project, including documentation updates. If we ever need to revert, how will we manage that given the way things are split?